fbchat/graphql.py

```diff
@@ -131,47 +131,55 @@ def graphql_to_attachment(a):
 def graphql_to_extensible_attachment(a):
     story = a.get('story_attachment')
     if story:
-        _type = story['target']['__typename']
-        if _type == 'MessageLocation':
-            latitude, longitude = get_url_parameter(get_url_parameter(story['url'], 'u'), 'where1').split(", ")
-            rtn = LocationAttachment(
-                uid=int(story['deduplication_key']),
-                latitude=float(latitude),
-                longitude=float(longitude),
-            )
-            rtn.image_url = story['media']['image']['uri']
-            rtn.image_width = story['media']['image']['width']
-            rtn.image_height = story['media']['image']['height']
-            rtn.url = story['url']
-            return rtn
-        elif _type == 'MessageLiveLocation':
-            rtn = LiveLocationAttachment(
-                uid=int(story['target']['live_location_id']),
-                latitude=story['target']['coordinate']['latitude'] if story['target'].get('coordinate') else None,
-                longitude=story['target']['coordinate']['longitude'] if story['target'].get('coordinate') else None,
-                name=story['title_with_entities']['text'],
-                expiration_time=story['target']['expiration_time'] if story['target'].get('expiration_time') else None,
-                is_expired=story['target']['is_expired'],
-            )
-            rtn.image_url = story['media']['image']['uri']
-            rtn.image_width = story['media']['image']['width']
-            rtn.image_height = story['media']['image']['height']
-            rtn.url = story['url']
-            return rtn
-        elif _type in ['ExternalUrl', 'Story']:
-            return ShareAttachment(
+        target = story.get('target')
+        if target:
```

Comment:
Is there nothing else that can determine whether the attachment is a `DeletedMessage`?

Reply:
No, just all information about the attachment is `null`